### DISCUSSION TITLE
Dịch step function

### DISCUSSION THREAD

**Discussion by cuongvng** [mlbvn/handson-ml2-vn#99] [Terminology] [answered]
Hàm bước nhảy

mn thấy sao? cc: @mlbvn/handson-ml

**Comment by minhduc0711** [comment, marked as answer]
`hàm bước` cho ngắn gọn được không?

**Reply by quangvnai** [reply]
Có hàm bước này: https://vi.wikipedia.org/wiki/H%C3%A0m_b%C6%B0%E1%BB%9Bc

**Reply by ngcthuong** [reply]
Từ này dịch nhiều trong sách dạy đại học rồi thì phải.

Bản Wiki @davidnvq trích dẫn không có nguồn tiếng Việt.
